gpytorch/utils/permutation.py

```diff
@@ -76,7 +76,8 @@ def apply_permutation(
         right_permutation = torch.arange(matrix.size(-1), device=matrix.device)
 
     # Apply permutations
-    return delazify(matrix.__getitem__((*batch_idx, left_permutation.unsqueeze(-1), right_permutation.unsqueeze(-2))))
+    res = delazify(matrix.__getitem__((*batch_idx, left_permutation.unsqueeze(-1), right_permutation.unsqueeze(-2))))
```

Comment:
Sorry about that, still getting used to pull requests and pre-commit. Don't think the problem was with this line in `permutation.py` (this one should be 118 long). I think the issue was with my first commit because I resolved the changes, but did not actually push a fix.